sa_tests.ml

```diff
@@ -0,0 +1,31 @@
+(*IMPORTANT (#use "compiler.ml";;) must be called before calling (#use "sa_tests.ml";;)*)
+
```

Comment:
```suggestion
#use "compiler.ml";;
```
אם אתה תלוי בזה, תייבא את זה. 
תמנע כמה שיותר מטעויות משתמש.

Reply:
The problem was double #use from sa_tests.ml and tester.ml, but it seems that it works.
Approved.